01_financial-well-being/housing-value.qmd

```diff
@@ -0,0 +1,904 @@
+---
+title: "Housing Value"
+author: " Manu Alcalá Kovalski"
+date: now
+format:
+  html:
+    embed-resources: true
+    toc: true
+    toc_float: true
+    code-fold: show
+    code-tools: true
+editor_options: 
+  chunk_output_type: console
+execute:
+  warning: false
+---
+
+```{=html}
+<style>
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
+</style>
+```
+<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato" />
+
+This program creates the **Housing Value** metric defined as the ratio between the share of a community’s housing wealth held by a racial or ethnic group to the share of households of the same group metric
+
+Initially written by Jung Hyun Choi and updated by Manu Alcalá Kovalski in 2023-2024.
+
+## Process
+
+-   [Housekeeping](#housekeeping)
+-   [Read Data](#read-data)
+-   [Clean Data](#clean-data)
+-   [Merge Crosswalk](#merge-crosswalk)
+-   [Create Housing Value Metric](#create-metric)
+-   [Evaluate Results](#evaluate-results)
+-   [Export](#export)
+
+## Housekeeping {#sec-housekeeping}
+
+```{r}
+#| label: setup
+
+options(scipen = 999)
+librarian::shelf(tidyverse, ipumsr, here, glue, labelled, skimr, urbnthemes, tidylog, assertr)
+set_urbn_defaults(style = "print")
+```
```

Comment:
For folks who don't have urbnthemes loaded, line 46 will not work. Recommend removing or including instructions for download, as librarian::shelf will not work for this package.